### PR TITLE
2 CLB Cards

### PR DESCRIPTION
**Pull request by JohnWilliams77**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/patriars_seal.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Patriar's Seal
+ManaCost:3
+Types:Artifact
+A:AB$ Mana | Cost$ T | Produced$ Any | SpellDescription$ Add one mana of any color.
+A:AB$ Untap | Cost$ 1 T | ValidTgts$ Creature.Legendary+YouCtrl | SpellDescription$ Untap target legendary creature you control.
+Oracle:{T}: Add one mana of any color.\n{1}, {T}: Untap target legendary creature you control.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/patriars_seal.txt:6]: `DeckHints:Type$Legendary`

File: forge-gui/res/cardsfolder/upcoming/prized_statue.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Prized Statue
+ManaCost:2
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield or is put into a graveyard from the battlefield, create a Treasure token.
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigToken | Secondary$ True | TriggerDescription$ When CARDNAME enters the battlefield or is put into a graveyard from the battlefield, create a Treasure token.
+SVar:TrigToken:DB$ Token | TokenScript$ c_a_treasure_sac
+DeckHas:Ability$Token
+Oracle:When Prized Statue enters the battlefield or is put into a graveyard from the battlefield, create a Treasure token. (It's an artifact with “{T}, Sacrifice this artifact: Add one mana of any color.”)
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/prized_statue.txt:8]: `“` -> `"`
`”` -> `"`
Also, please add this reminder parenthetical to the TriggerDescription in line 4

**Review by Northmoc — COMMENTED** [review]
Looks good; just a few tweaks for you!

File: forge-gui/res/cardsfolder/upcoming/prized_statue.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Prized Statue
+ManaCost:2
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield or is put into a graveyard from the battlefield, create a Treasure token.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/prized_statue.txt:4]: please add `(It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")` to TriggerDescription

you can also add it to line 5 if you like (your call)

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/prized_statue.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Prized Statue
+ManaCost:2
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield or is put into a graveyard from the battlefield, create a Treasure token.(It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
```
[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/prized_statue.txt:4]: The one between . and " is wrong now though, sorry

**Pull request merged**